Database.py

```diff
@@ -114,6 +115,7 @@ def exportDB(self, exportPath: Path = Path(""), adminInfo: Boolean = False, show
         with open(exportPath, 'w', newline='') as csvfile:
             fieldnames = dir(EmployeeContainer)
             badnames = ['permissionList', 'getFName', 'getLName']
+            badnames = badnames if adminInfo else adminFields
```

Comment:
I think should be if not adminInfo else badnames+adminFields

Reply:
I can change that, right now I had that included in the admin fields, but that makes sense